### ISSUE TITLE
Writing varargs `T...` instead `...: T`

### ISSUE THREAD

**Issue by UrNightmaree** [teal-language/tl#671]
Why don't we make varargs typing to `T...` instead `...: T`, especially in record or enum fields?
```lua
local record someMod
    modFunc: function(string...): string
end
```

**Comment by 13k** [comment]
I think it's already implemented in 90ef2480434adfcf103dd9ec84c7c4b2ea9e77c2

**Comment by hishamhm** [comment]
Indeed!
